openlibrary/plugins/upstream/models.py

```diff
@@ -323,7 +323,8 @@ def set_identifiers(self, identifiers):
             name, value = id['name'], id['value']
             if name == 'lccn':
                 value = normalize_lccn(value)
-            d.setdefault(name, []).append(value)
+            if value is not None:
```

Comment:
Some comment about "we should surface this warning/error to the user might be nice." linked to an issue.

Reply:
I have added such language.